pkg/operator/k8sutil/node.go

```diff
@@ -329,33 +329,41 @@ func RookNodesMatchingKubernetesNodes(rookStorage rookv1.StorageScopeSpec, kuber
 }
 
 // ExtractTopologyFromLabels extracts rook topology from labels and returns a map from topology type to value
```

Comment:
While we are here, it would be good to update this doc to explain that additionalTopologyIDs should have a specific ordering from lowest to highest hierarchy and that they should be of the form "rack" (not "topology.rook.io/rack"). And update to explain the new return value.